lib/inherited_resources/base_helpers.rb

```diff
@@ -226,14 +226,14 @@ def get_resource_ivar #:nodoc:
       # Set resource ivar based on the current resource controller.
       #
       def set_resource_ivar(resource) #:nodoc:
-        instance_variable_set("@#{resource_instance_name}", resource)
+        instance_variable_set(:"@#{resource_instance_name}", resource)
       end
 
       # Get collection ivar based on the current resource controller.
       #
       def get_collection_ivar #:nodoc:
         if instance_variable_defined?(:"@#{resource_collection_name}")
-          instance_variable_get("@#{resource_collection_name}")
+          instance_variable_get(:"@#{resource_collection_name}")
```

Comment:
guess this wasn't covered before

Reply:
Yes, that's right. I had encountered the same on a work project. It was a reported fix in the release notes. Thanks!